python/cugraph/cugraph/tests/link_prediction/test_jaccard.py

```diff
@@ -153,6 +154,54 @@ def networkx_call(M, benchmark_callable=None):
     return src, dst, coeff
 
 
+# FIXME: This compare is shared across several tests... it should be
+#        a general utility
+def compare(src1, dst1, val1, src2, dst2, val2):
```

Comment:
It looks like `compare` is always being passed values as 3 individual series from a dataframe.  Since `compare` will just re-create these as a dataframe, can compare be written to be called like this: `compare(all_pairs_jaccard_results, jaccard_results, ['first', 'second', 'jaccard_coeff'])` ?
```
def compare(a, b, names):
    df1 = a[names]
    df2 = b[names]
    join = df1.merge(...)
    ...
```

Reply:
I chatted with @ChuckHastings offline - I'll take care of refactoring this test utility in a separate PR since I'm looking at updates like this anyway.

Reply:
I'm addressing much of this feedback in https://github.com/rapidsai/cugraph/pull/4776
I'm not rewriting as much of the `compare` utility as I proposed since I can't tell how general purpose it was intended to be (ex. it currently supports passing in arrays, should that still be allowed?)